### PR TITLE
fix PrimaryKey on circles_event

### PR DESCRIPTION
**Pull request by ArtificialOwl**

### CODE REVIEW

File: lib/Migration/Version0024Date20220203123901.php [hunk 1]
```diff
@@ -39,11 +39,11 @@
 use OCP\Migration\SimpleMigrationStep;
 
 /**
- * Class Version0023Date20211216113101
+ * Class Version0024Date20220203123901
```
Comment (ArtificialOwl) [inline lib/Migration/Version0024Date20220203123901.php:42]: keep an empty version of `Version0023Date20211216113101`

File: lib/Migration/Version0024Date20220203123902.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+						 ]
+			);
+			$table->addColumn(
+				'creation', 'bigint', [
+							  'length' => 14,
+							  'notnull' => false
+						  ]
+			);
+
+			$table->addUniqueIndex(['token', 'instance']);
```
Comment (ArtificialOwl) [inline lib/Migration/Version0024Date20220203123902.php:124]: primaryKey !?

Comment (CarlSchwan) [reply lib/Migration/Version0024Date20220203123902.php:124]: It would make sense ihmo

File: lib/Migration/Version0024Date20220203123902.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+class Version0024Date20220203123902 extends SimpleMigrationStep {
+
+
+	/**
+	 * @param IDBConnection $connection
+	 */
+	public function __construct(IDBConnection $connection) {
+	}
+
```
Comment (CarlSchwan) [inline lib/Migration/Version0024Date20220203123902.php:49]:
```suggestion
```

not used

**Review by CarlSchwan — APPROVED** [review]

File: lib/Migration/Version0024Date20220203123901.php [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if ($schema->hasTable('circles_token')) {
+			$table = $schema->getTable('circles_token');
+			$table->changeColumn(
+				'password', [
+							  'length' => 127
+						  ]
```
Comment (CarlSchwan) [inline lib/Migration/Version0024Date20220203123901.php:73]:
```suggestion
						'length' => 127
					]
```

**Pull request merged**